### PR TITLE
Fullcalendar io

### PR DESCRIPTION
**Pull request by daniel-samson**
Adding new calendar to SuiteCRM that @ijdavie worked on. It uses the [fullcalendar](http://fullcalendar.io/) library

## Description
The new calendar is more responsive, and add views which better displays shared activities for different users. Users can customise the colour of the activities. 

Also the new calendar uses bootstrap modal dialog forms to display settings and to create or edit activities.

## Motivation and Context

- It's a step to begin moving away from the YUI calendar, and more towards jQuery.
- It gives a better user experience.

## How To Test This

- Test the celendar, meetings, calls dashboard.
- Test freebusy functionality (vCard, schedule bar)
- Test the calendar module. 

## Types of changes
- Feature: (new views in the calendar)
- Feature: Drag and drop
- Feature: color pickers in settings.
- Feature: Modal dialog

### Final checklist

- [ ] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [ ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Calendar/tpls/shared_users.tpl [hunk 1]
```diff
@@ -102,37 +102,61 @@
...
-</div>
+<div class="modal fade modal-calendar-user-list" tabindex="-1" role="dialog">
+	<div class="modal-dialog modal-lg">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
+				<h4 class="modal-title">{$MOD.LBL_EDIT_USERLIST}</h4>
+			</div>
+			<div class="modal-body">
+				<!--->
```
Comment (samus-aran) [inline modules/Calendar/tpls/shared_users.tpl]: Alot of commentng out

File: themes/SuiteP/modules/Calendar/tpls/footer.tpl [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{*
+/*********************************************************************************
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
+ * Copyright (C) 2011 - 2014 Salesagility Ltd.
```
Comment (samus-aran) [inline themes/SuiteP/modules/Calendar/tpls/footer.tpl]: Update License

File: themes/SuiteP/modules/Calendar/tpls/form.tpl [hunk 3]
```diff
@@ -0,0 +1,95 @@
+{*
+/*********************************************************************************
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
+ * Copyright (C) 2011 - 2014 Salesagility Ltd.
```
Comment (samus-aran) [inline themes/SuiteP/modules/Calendar/tpls/form.tpl]: Update License

File: themes/SuiteP/modules/Calendar/tpls/header.tpl [hunk 4]
```diff
@@ -0,0 +1,97 @@
+{*
+/*********************************************************************************
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
+ * Copyright (C) 2011 - 2014 Salesagility Ltd.
```
Comment (samus-aran) [inline themes/SuiteP/modules/Calendar/tpls/header.tpl]: Update License

File: themes/SuiteP/modules/Calendar/tpls/settings.tpl [hunk 5]
```diff
@@ -0,0 +1,224 @@
+{*
+/*********************************************************************************
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
+ * Copyright (C) 2011 - 2014 Salesagility Ltd.
```
Comment (samus-aran) [inline themes/SuiteP/modules/Calendar/tpls/settings.tpl]: Update License

File: themes/SuiteP/modules/Calendar/tpls/settings.tpl [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+<div class="modal fade modal-calendar-settings" tabindex="-1" role="dialog">
+	<div class="modal-dialog modal-lg">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
+				<h4 class="modal-title">{$MOD.LBL_SETTINGS_TITLE}</h4>
+			</div>
+			<div class="modal-body">
+				<!--->
+				<div class="container-fluid">
```
Comment (samus-aran) [inline themes/SuiteP/modules/Calendar/tpls/settings.tpl]: Again - too much commented out

File: themes/SuiteP/modules/Calendar/tpls/shared_users.tpl [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+<div class="modal fade modal-calendar-user-list" tabindex="-1" role="dialog">
+	<div class="modal-dialog modal-lg">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
+				<h4 class="modal-title">{$MOD.LBL_EDIT_USERLIST}</h4>
+			</div>
+			<div class="modal-body">
+				<!--->
```
Comment (samus-aran) [inline themes/SuiteP/modules/Calendar/tpls/shared_users.tpl]: Remove rather than commented out

**Comment by horus68** [comment]
@daniel-samson Is this PR included in the Master branch for 7.9.0 ?

**Comment by chris001** [comment]
@horus68 It's included in there now: https://github.com/salesagility/SuiteCRM/tree/master/modules/Calendar/fullcalendar

**Pull request merged**